pip_executable_choco/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,89 @@
+﻿function Get-RegistryValue($key, $value) {
+  $item = (Get-ItemProperty $key $value -ErrorAction SilentlyContinue)
+  if ($item -ne $null) { return $item.$value } else { return $null }
+}  
+
+function Get-Python-Home() {
+  $result = $null
+  
+  $filename = Get-RegistryValue "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\Python.exe" '(default)' 
+  
+  if ($null -eq $filename) {
```

Comment:
Could be rewritten simpler as `if (!$filename) {`... is there a reason why not to do that?

(This comment applies elsewhere here.)

Reply:
> is there a reason why not to do that?

Just that I didn't know PS allowed that.